web-app/src/pages/Login.vue

```diff
@@ -50,6 +52,8 @@ import axios, { AxiosResponse } from 'axios';
 
 @Component
 export default class Login extends Vue {
+  private logoUrl: unknown = '';
```

Comment:
```suggestion
  private logoUrl: string = '';
```

Reply:
This way TS throws an error. Tried it.

Reply:
Change this after updated requests from #10. Then it will be okay